CHANGELOG.md

```diff
@@ -1,4 +1,80 @@
-## 2.19.4 (Unreleased)
+## 2.19.5 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.4**
+- Supported Terraform version: **v1.x**
+
+### Features:
+#### Accounts
+1. Implemented support for AWSTS in **aviatrix_account** and data source with the following new attributes:
+- ``awsts_account_number``
+- ``awsts_cap_url``
+- ``awsts_cap_agency``
+- ``awsts_cap_mission``
+- ``awsts_cap_role_name``
+- ``awsts_cap_cert``
+- ``awsts_cap_cert_key``
+- ``awsts_ca_chain_cert``
+2. Implemented support for AWSS in **aviatrix_account** and data source with the following new attributes:
+- ``awss_account_number``
+- ``awss_cap_url``
+- ``awss_cap_agency``
+- ``awss_cap_account_name``
+- ``awss_cap_role_name``
+- ``awss_cap_cert``
+- ``awss_cap_cert_key``
+- ``awss_ca_chain_cert``
+
+#### Firewall Network
+1. Implemented support for Fail Close and Network List Excluded From East-West Inspection in **aviatrix_firenet**
+2. Implemented support for colon character in ``tags`` in **aviatrix_firewall_instance**
```

Comment:
As Matthew has stated previously, colons are already supported. Perhaps he can provide a better description of the json tag change. This change is supposed to be completely transparent to user workflow